cibyl/cli/parser.py

```diff
@@ -166,7 +188,7 @@ def extend(self, arguments: List[Argument], group_name: str,
         # If the group doesn't exists, we would like to add it
         # so arguments are grouped based on the model class they belong to
         if not group:
-            group = self.argument_parser.add_argument_group(group_name)
+            group = self.model_parser.add_argument_group(group_name)
```

Comment:
I see both `model_parser` and `app_parser` have the same `group` variable name, maybe it could be nice to change both to something more descriptive.

Reply:
In both cases the `group` variable is a local variable that refers to a new argparse argument group object. To avoid any confusion, I've renamed the `group` variable from the `__add_arguments` method to something more explicit.

Reply:
Perfect. It looks great. Thank you, Joan!